chartist-plugin-legend.js

```diff
@@ -36,6 +42,18 @@
             return a - b;
         }
 
+        // Parse input position string to enum
```

Comment:
Is using an enum really necessary? Why not only allow `"bottom"` and `"top"` as a string, and throw an exception if it's not one of the two.